src/routes/resources/[resourceContentId]/+page.svelte

```diff
@@ -233,7 +233,14 @@
             new Map(resourceContent.machineTranslations.map((mt) => [mt.contentIndex, mt]))
         );
         promptForMachineTranslationRating.set(
-            resourceContent.machineTranslations.some((mt) => !mt.userRating && currentUserIsAssigned)
+            resourceContent.machineTranslations.some(
+                (mt) =>
+                    !mt.userRating &&
+                    currentUserIsAssigned &&
+                    resourceContent?.status !== ResourceContentStatusEnum.TranslationPublisherReview &&
```

Comment:
We only want this in EditorReview status, so wouldn't we just check against that, rather than three things it isn't?

Reply:
Makes sense. I'll put in another PR for the change